URL-API.md

```diff
@@ -4,77 +4,289 @@ MazeMap URL API
-To start mazemap with a generic point in view use longitude and lattitude format
+#### Specifying a generic point destination
+
+To start MazeMap with a generic point in view:
 
 ```
 http://use.mazemap.com/?campusid=1&desttype=point&dest=10.4026794,63.4183615,0
 http://use.mazemap.com/?campusid=1&desttype=point&dest=10.4035833,63.4178412,3
 ```
-*   _**desttype**_ defines the type expected for the dest parameter. For a generic point, you can must use the word **point** here.
-*   _**dest**_ defines the geographical point in 4326 comma-separated array as shown below.
-```javascript
-dest=lon,lat,z
-``` 
-The the z value is used to specify the _floor_, if the point lies inside a building. If outside, the _**z**_ parameter can be dropped, or be simply 0.
 
+* `desttype` defines the format expected by the `dest` parameter. In this case a generic point.
+* `dest` defines the geographical point as a comma-separated list `dest=longitude,latitude,z` where `z` is used to specify the _floor_ if the point lies inside a building. If outside, the `z` parameter should be 0, or it can simply be dropped.
 
 
-4. Defining a path
+Defining a path
 --------------------
-To start mazemap with a predefined path, use the _**starttype**_ and _**start**_ parameters along with the previous _**desttype**_ and _**dest**_.
+
+To start MazeMap with a predefined path:
 
 ```
-http://use.mazemap.com/?campusid=1&desttype=poi&dest=35994&starttype=point&start=10.4029047,63.4186015,0
+http://use.mazemap.com/?campusid=1&starttype=point&start=10.4029047,63.4186015,0&desttype=poi&dest=35994
 ```
-*   _**starttype**_ defines the type expected for the start parameter. In the example, a geographical _point_ outside is used, as described earlier.
-*   _**start**_ defines the geographical point in 4326 comma-separated array as described earlier. Can also be a POI.
-*   _**desttype**_ In the example, a POI is used as desttype.
-*   _**dest**_ In the example the poiID for MazeMap office is used as destination.
+
+* `starttype` defines the format expected by the `start` parameter. In this case a geographical point outside.
+* `start` defines the geographical point in a comma-separated list.
+* `desttype` In the example, a POI is used as destination type.
+* `dest` In the example the poiID for MazeMap office is used as destination.
 
 
-5. Defining a custom name for start/destination
+Defining custom names for points
 -----------------------------------------------
-When starting MazeMap with predefined start or destination, you can input a custom string for the names of these points. I.e. if you you are linking to an office with id 404, you can specify that it should display as 'John's Office'.
 
+When starting MazeMap either with predefined points (e.g. if you have defined a path), you can input a custom string for the names of these points. I.e. if you you are linking to an office with id 404, you can specify that it should display as "John's Office". Use `%20` for spaces.
```

Comment:
It is not needed to replace spaces by `%20`s. I would expect users of this API to know about URLencoding, how to do it, and when to do it. In fact, proper URL encoding means that commas in the geographical coordinates should also be encoded into `%2C`s.